### PR TITLE
Add rubocop-rspec

### PR DESCRIPTION
**Pull request by teceler**

### CODE REVIEW

**Comment by teceler** [comment]
I have two remaining cop complaints that codeclimate is apparently not catching -- possibly because it doesn't like checking tests -- documenting here:

```
spec/controllers/api/v1/characters_controller_spec.rb:259:5: C: RSpec/RepeatedDescription: Don't repeat descriptions within an example group.
    it "sets correct alias_id_for_post when given post_id with recently used alias in post", :show_in_doc do
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

spec/controllers/api/v1/characters_controller_spec.rb:269:5: C: RSpec/RepeatedDescription: Don't repeat descriptions within an example group.
    it "sets correct alias_id_for_post when given post_id with recently used alias in post" do
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

File: spec/controllers/board_sections_controller_spec.rb [hunk 1]
```diff
@@ -101,6 +101,7 @@
     end
 
     it "works with login" do
+      login
```
Comment (Throne3d) [inline spec/controllers/board_sections_controller_spec.rb]: Oops.

File: spec/controllers/characters_controller_spec.rb [hunk 2]
```diff
@@ -594,16 +594,6 @@
       expect(character.galleries).to eq([])
     end
 
-    it "creates new templates when specified" do
```
Comment (Throne3d) [inline spec/controllers/characters_controller_spec.rb]: What's up with this?

Comment (teceler) [reply spec/controllers/characters_controller_spec.rb]: It's a duplicate of another test. I can find the test again, just a sec

Comment (teceler) [reply spec/controllers/characters_controller_spec.rb]: https://github.com/Marri/glowfic/blob/05d3a49010d82a6183c00eb58d253c5104491cdf/spec/controllers/characters_controller_spec.rb#L505 and https://github.com/Marri/glowfic/blob/05d3a49010d82a6183c00eb58d253c5104491cdf/spec/controllers/characters_controller_spec.rb#L597

Comment (Throne3d) [reply spec/controllers/characters_controller_spec.rb]: Ah, cool!

**Comment by Throne3d** [comment]
Looks good to merge when you're happy with it, then!

**Pull request merged**